core/domain/state_domain.py

```diff
@@ -281,8 +281,8 @@ def __init__(
             interaction_id: str. The interaction id.
             answer_is_exclusive: bool. True if is the only correct answer;
                 False if is one of possible answer.
-            correct_answer: str. The correct answer; this answer enables the
-                learner to progress to the next card.
+            correct_answer: BaseInteraction.answer_type. The correct answer;
```

Comment:
The correct typeinfo should be *. Explain how * is determined in the description.

"BaseInteraction.answer_type" is not a type. The typeinfo should be what you get when you call type() on the arg.

Reply:
done